src/modules/Header/components/LinkItemUpper.js

```diff
@@ -35,6 +36,9 @@ const LinkItemUpper = styled(NavLink)`
   
   &.has-separator {
     border-right: 1px solid ${COLORS.OIVA_MENU_BG_COLOR};
+    @media ${MEDIA_QUERIES.MOBILE} {
```

Comment:
The implementation can be removed if we use Tailwind's responsive classes and modular border definition.